src/main/java/org/zalando/nakadi/service/subscription/SubscriptionService.java

```diff
@@ -214,10 +218,12 @@ public Result listSubscriptions(final String owningApplication, final Set<String
 
         return eventTypes.stream()
                 .map(eventType -> {
-                    final Set<SubscriptionEventTypeStats.Partition> statPartitions = Arrays.stream(partitions)
+                    final Set<SubscriptionEventTypeStats.Partition> statPartitions =
+                            Arrays.stream(zkSubscriptionNode.getPartitions())
+                            .sorted((p1, p2) -> p1.getKey().getPartition().compareTo(p2.getKey().getPartition()))
```

Comment:
Just reminded we did not padded it... it almost hurt ourselves. Added a ticket to remind.